### PR TITLE
[1] Delegators endpoint 

### PR DESCRIPTION
**Pull request by eesmerdag**
https://www.notion.so/figmentnetworks/Near-Indexer-Track-delegations-per-epoch-3774ee3f9b584e32accca45c88b434fe

### CODE REVIEW

File: model/mapper/delegation.go [hunk 1]
```diff
@@ -25,10 +33,28 @@ func Delegations(input []near.AccountInfo) ([]model.Delegation, error) {
 func Delegation(input *near.AccountInfo) (*model.Delegation, error) {
 	delegation := &model.Delegation{
 		Account:         input.Account,
-		UnstakedBalance: types.NewAmount(input.StakedBalance),
+		UnstakedBalance: types.NewAmount(input.UnstakedBalance),
```
Comment (pacmessica) [inline model/mapper/delegation.go:36]: oi, bug. So is `Delegation` saved to db? Do we need to resync and correct this value?

File: server/server.go [hunk 2]
```diff
@@ -449,6 +450,25 @@ func (s Server) GetDelegations(c *gin.Context) {
 	jsonOk(c, delegations)
 }
 
+// GetDelegators returns list of delegators
+func (s Server) GetDelegators(c *gin.Context) {
+	search := store.DelegatorEpochsSearch{}
+	search.Epoch = c.Query("epoch")
+	search.ValidatorID = c.Param("id")
```
[the remark this answers is not in the document]
Comment (pacmessica) [reply server/server.go:457]: > Why not bind this search input instead?

not sure what you mean by this. But agree `search.ValidatorID = c.Param("id")` doesn't do anything. Maybe just make it a query "validator_id"

[the remark this answers is not in the document]
Comment (sosedoff) [reply server/server.go:457]: I was referring to `router.GET("/delegators", s.GetDelegators)` part `c.Param(..)` will pick up params from the url and not from the query params.

[the remark this answers is not in the document]
Comment (eesmerdag) [reply server/server.go]: corrected 👍

File: server/server.go [hunk 3]
```diff
@@ -449,6 +450,25 @@ func (s Server) GetDelegations(c *gin.Context) {
 	jsonOk(c, delegations)
 }
 
+// GetDelegators returns list of delegators
+func (s Server) GetDelegators(c *gin.Context) {
+	search := store.DelegatorEpochsSearch{}
```
Comment (sosedoff) [inline server/server.go:455]: why cant we use `BindQuery` in here like with other types of search requests?

**Review by sosedoff — APPROVED** [review]
Looks good - just had one question about the query binding.

**Review by pacmessica — APPROVED** [review]

**Pull request merged**